src/BenchmarkDotNet/Toolchains/MonoWasm/WasmToolChain.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using JetBrains.Annotations;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Loggers;
+
+
+
+
+namespace BenchmarkDotNet.Toolchains.MonoWasm
+{
+    [PublicAPI]
+    public class WasmToolChain : Toolchain
+    {
+        [PublicAPI] public static readonly IToolchain NetCoreApp50Wasm = From(WasmAppSettings.NetCoreApp50Wasm);
+
+        private WasmToolChain(string name, IGenerator generator, IBuilder builder, IExecutor executor, string runtimeJavaScriptPath)
+            : base(name, generator, builder, executor)
+        {
+        }
+
+        public WasmToolChain(string name,
+                             string targetFrameworkMoniker,
+                             string cliPath,
+                             string packagesPath,
+                             string runtimeFrameworkVersion,
```

Comment:
if `runtimeFrameworkVersion` makes no sense for `Wasm`, we can just remove it and make `WasmGenerator` pass `null` to base ctor (the same applies to other unused settings)